pkg/spec/spec.go

```diff
@@ -39,6 +39,7 @@ type FunctionSpec struct {
 	Deps     string            `json:"deps"`
 	Memory   resource.Quantity `json:"memory"`
 	Env      map[string]string `json:"env"`
+	Desc     string            `json:"description"`
```

Comment:
In the issue https://github.com/kubeless/kubeless/issues/215 I mentioned that the description should be an annotation of the function object. So to me it should be put in the metadata just like the name and the labels.